packages/babel-helper-check-duplicate-nodes/README.md

```diff
@@ -1,14 +1,15 @@
-# babel-check-duplicated-nodes
```

Comment:
Can you exclude this file from https://github.com/babel/babel/blob/main/scripts/generators/readmes.js, so that it doesn't get overwritten?